src/utils/index.js

```diff
@@ -52,3 +52,10 @@ export function unprefixKeys(obj, prefixToRemove, lowercaseFirst = true) {
     return prev;
   }, {});
 }
+
+/**
+ * Returns a className string containing all non-empty classes from the given list of candidates.
+ */
+export function joinClasses(...candidates) {
```

Comment:
Are there any edge cases with a custom implementation of this vs what React recommends here? https://github.com/JedWatson/classnames